data-tck/src/main/java/io/micronaut/data/tck/entities/Meal.java

```diff
@@ -26,14 +26,13 @@
 import java.util.Date;
 import java.util.HashSet;
 import java.util.Set;
-import java.util.UUID;
 
 @MappedEntity
 public class Meal {
 
     @Id
-    @AutoPopulated
-    private UUID mid;
+    @GeneratedValue
+    private Long mid;
```

Comment:
The [tests for the locking behavior](https://github.com/micronaut-projects/micronaut-data/pull/738/files#diff-ccd012a0a1b9364d79e56fc2fb77ed9aR930) don't work for MariaDB, MySQL and Oracle when using `UUID`. In some cases one of the threads doesn't update the record, but even when both of them do, the record stays unchanged when it's fetched again in the assertions. I couldn't figure out why. When using `Long` everything is fine, though.